docs/modules/ROOT/pages/reference/extensions/quarkus-cxf-rt-transports-http-hc5.adoc

```diff
@@ -67,7 +67,7 @@ in the source tree of {quarkus-cxf-project-name}
 include::example$hc5/async-binding.xml[]
 ----
 
-This file should then be passed to xref:user-guide/contract-first-code-first/generate-java-from-wsdl.adoc[wsdl2java]
+This file should then be passed to xref:user-guide/contract-first-code-first/generate-java-from-wsdl.adoc#generate-java-from-wsdl[wsdl2java]
```

Comment:
This needs to get changed in `extensions/transports-http-hc5/runtime/src/main/doc/usage.adoc`